src/client/Microsoft.Identity.Client/ApiConfig/AcquireTokenByAuthorizationCodeParameterBuilder.cs

```diff
@@ -66,6 +66,24 @@ internal override Task<AuthenticationResult> ExecuteInternalAsync(CancellationTo
         {
             return ConfidentialClientApplicationExecutor.ExecuteAsync(CommonParameters, Parameters, cancellationToken);
         }
+
+        /// <summary>
+        /// Specifies if the x5c claim (public key of the certificate) should be sent to the STS.
+        /// Sending the x5c enables application developers to achieve easy certificate roll-over in Azure AD:
+        /// this method will send the public certificate to Azure AD along with the token request,
+        /// so that Azure AD can use it to validate the subject name based on a trusted issuer policy.
+        /// This saves the application admin from the need to explicitly manage the certificate rollover
+        /// (either via portal or powershell/CLI operation)
+        /// </summary>
+        /// <param name="withSendX5C"><c>true</c> if the x5c should be sent. Otherwise <c>false</c>.
+        /// The default is <c>false</c></param>
+        /// <returns>The builder to chain the .With methods</returns>
+        public AcquireTokenByAuthorizationCodeParameterBuilder WithSendX5C(bool withSendX5C)
```

Comment:
Why don't we move this to the ConfidentialClientParameterBuilder?

Reply:
Putting it in the abstract builder will require some additional work to expose the child parameter property to the parent abstract class. 

referring to line 84
"Parameters.SendX5C = withSendX5C;"

Reply:
since this is time sensitive and needs to be shipped very soon I just want to implement the quick fix now. I can create another task to clean up/refactor the code at a later point.